tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1665,4 +1665,70 @@ public function theAdministratorTriesToAddTheFollowingUsersToAGroupAtOnceUsingTh
 		}
 		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
 	}
+
+	/**
+	 * @When user :user gets details of the group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
+		$this->getDetailsOfGroup($user, $groupId);
```

Comment:
```suggestion
		$this->getDetailsOfGroup($user, $this->getGroupIdByName($groupName));
```
We can make this inline since we're not re-using the variable